airflow/__init__.py

```diff
@@ -74,11 +74,13 @@ def __getattr__(name):
 if not settings.LAZY_LOAD_PROVIDERS:
     from airflow import providers_manager
 
-    providers_manager.ProvidersManager().initialize_providers_manager()
+    manager = providers_manager.ProvidersManager()
+    manager.initialize_providers_list()
+    manager.initialize_providers_hooks()
+    manager.initialize_providers_extra_links()
 
 
 # This is never executed, but tricks static analyzers (PyDev, PyCharm,)
-# into knowing the types of these symbols, and what
 # they contain.
```

Comment:
@potiuk Was this line meant to be removed?

Reply:
Not at all - you can add PR to bring it back :).

Reply:
@potiuk Thanks! I opened #17884 to fix it.

Reply:
Merged! Quickest contribution EVER!

Reply:
Thanks! I did not expect my first contribution to be this quick 😂